js/app/src/lite/custom-element/index.ts

```diff
@@ -157,13 +152,24 @@ export function bootstrap_custom_element(
 			}
 
 			const codeElements = this.getElementsByTagName("gradio-code");
-			if (codeElements.length > 1) {
-				console.warn(
-					"Multiple <gradio-code> elements are found. Only the first one will be used."
-				);
+			if (codeElements.length === 0) {
+				// If there is no <gradio-code> element, try to parse the content of the custom element as code.
+				let code = "";
+				this.childNodes.forEach((node) => {
+					if (node.nodeType === Node.TEXT_NODE) {
+						code += node.textContent;
+					}
+				});
+				options.code = code.trim() || undefined;
```

Comment:
Why are you using `.trim` here? 

I think it's conflicting with my `clean_indent` function which is causing a lot of indentation issues. See the index.html file. 

https://github.com/gradio-app/gradio/blob/06bdf0eddf59bf79b6b4d1268fc9290955ef2490/js/app/src/lite/index.ts#L93-L111 

I would just remove it, but let me know what you think

Reply:
Sorry I just added it for defensively cleaning up the code without being aware of the side effect.
Just remove it.